tslint.json

```diff
@@ -10,6 +10,7 @@
         "no-internal-module": true,
         "no-trailing-whitespace": true,
         "no-var-keyword": true,
+        "no-unused-variable": true,
```

Comment:
Hm, gulp-typescript doesn't fail on type errors. I think it should, let's see if that's an option...